### DISCUSSION TITLE
Error while installing or adding any package (fatal: failed to load library 'libcurl-4.dll')

### DISCUSSION THREAD

**Discussion by arturbikbaev** [LykosAI/StabilityMatrix#294] [Q&A]
Hi Folks,

Just can't install or add any package to StabilityMatrix.

Getting the following everytime:

_Error: StabilityMatrix.Core.Exceptions.ProcessException: Process git exited with code 128. {StdOut = , StdErr = Cloning into 'C:\Users\РђСЂС‚СѓСЂ\AppData\Roaming\StabilityMatrix\Packages\stable-diffusion-webui'...
**fatal: failed to load library 'libcurl-4.dll'**_

I'm just wondering could the username folder in Cyrillic be the reason? Would appreciate any ideas and hints. Thanks.

Please find the screenshots below:

![Снимок экрана 2023-11-27 112238](https://github.com/LykosAI/StabilityMatrix/assets/16808007/4c9188d1-fbc2-4600-88a4-3f3819e7569e)
![Снимок экрана 2023-11-27 112245](https://github.com/LykosAI/StabilityMatrix/assets/16808007/43659e1b-a5c5-469a-9cb0-e857f9ab4d8a)

**Comment by arturbikbaev** [comment]
Update: the issue was solved after renaming in English the User Profile Folder Name Using the Registry Editor. 

So the username folder in Cyrillic was the reason and the problem. Could you please add support for Cyrillic User Profile Folder Names?

**Reply by Rodrigocglez** [reply]
i just started to use Github and tried to dm you but i dont know how lol, can you help me with that ? i have the same issue but i dont know how to solve it

**Comment by NdlelaYami** [comment]
Hi. I'm getting  a similar error but I'm sure my folder is using standard english characters:

New to SD. I can't seem to install any packages. I get this error:

Unpacking resources
Unpacking resources
Error: StabilityMatrix.Core.Exceptions.ProcessException: Process git exited with code 128. {StdOut = , StdErr = Cloning into 'C:\Users\thoug\AppData\Roaming\StabilityMatrix\Packages\ComfyUI'...
fatal: unable to access 'https://github.com/comfyanonymous/ComfyUI/': schannel: next InitializeSecurityContext failed: Unknown error (0x80092013) - The revocation function was unable to check revocation because the revocation server was offline.
}
at StabilityMatrix.Core.Processes.ProcessResult.EnsureSuccessExitCode()
at StabilityMatrix.Avalonia.Helpers.WindowsPrerequisiteHelper.RunGit(ProcessArgs args, String workingDirectory)
at StabilityMatrix.Core.Models.Packages.BaseGitPackage.DownloadPackage(String installLocation, DownloadPackageVersionOptions versionOptions, IProgress1 progress) at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IReadOnlyList1 steps)

Any suggestions on how to fix?

**Comment by BrechtCorbeel** [comment]
I got it to work after turning off output sharing in StabilityMatrix as a prerequisite before install.

**Reply by Yatela** [reply]
Hi! how u do that? i have the same problem

**Comment by danielhamelberg** [comment]
This worked for me:

Go to Settings | Package Environment 
Run a git command. Paste this in the popup: 
`config --global http.sslBackend openssl`
Now try installing a package again.

**Reply by neires** [reply]
Thank you very much!
That solved the problem for me.
